packages/ckeditor5-dev-utils/lib/bundler/geteditorconfig.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const javascriptStringify = require( 'javascript-stringify' );
+
+/**
+ * Transforms specified configuration to a string which can be inserted
+ * to generated an entry file.
+ *
+ * Keys and values will be wrapped in apostrophes instead of quotation marks.
```

Comment:
I wanted to comment on this documentation previously that it goes into implementation details rather than saying what it tries to achieve (which is – printing the object in our code style).

And now, notice how much outdated this documentation is after you changed the implementation ;)